### ISSUE TITLE
Use Base58 for API Keys

### ISSUE THREAD

**Issue by nelsonic** [dwyl/auth#49]
At present the `AUTH_API_KEY` PR #42  is using `exbase58` for Base58 encoding/decoding.
We need to: 
+ [x] replace it with https://github.com/dwyl/base58
+ [x] test that everything still works.

**Comment by nelsonic** [comment]
Sadly the https://github.com/dwyl/base58encode project is _one-way_ ...
We need the ability ***`decode`*** the encoded values so that we can ***`decrypt`*** them.
So we need to spend some time on https://github.com/dwyl/base58encode/issues/13 ⏳

**Comment by nelsonic** [comment]
I've added the ability to decode Base58 in https://github.com/dwyl/base58/pull/17 :shipit:

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/80292577-2ddf5a80-874f-11ea-84bd-4e9249c7ffe7.png)

**Comment by nelsonic** [comment]
@SimonLab once #43 is merged we can create a separate PR to replace `exbase58` (_temporary_) with `b58` (LTS). It should be a drop-in replacement as the OTP App has the same name.

**Comment by nelsonic** [comment]
Thanks for getting this done @SimonLab 👍
